core/shared/src/main/scala/org/http4s/headers/Origin.scala

```diff
@@ -28,48 +27,32 @@ sealed abstract class Origin
 object Origin {
   // An Origin header may be the string "null", representing an "opaque origin":
   // https://stackoverflow.com/questions/42239643/when-does-firefox-set-the-origin-header-to-null-in-post-requests
-  case object Null extends Origin
-
-  // If the Origin is not "null", it is a non-empty list of Hosts:
-  // http://tools.ietf.org/html/rfc6454#section-7
-  final case class HostList(hosts: NonEmptyList[Host]) extends Origin
+  case object `null` extends Origin
```

Comment:
I can't think of another case that we modeled the header as a sum type instead of a unary case class of a sum type, but I think I like it.  It sidesteps the `org.http4s.Origin` vs. `org.http4s.header.Origin` debate.  And it slipped through review the first time without me noticing!